junit-extensions/src/main/java/org/codice/junit/ClearInterruptions.java

```diff
@@ -22,7 +22,7 @@
 
 /**
  * The <code>ClearInterruptions</code> annotation can be used in conjunction with the {@link
- * org.codice.junit.MethodRuleAnnotationRunner} JUnit test runner or the {@link
+ * MethodRuleAnnotationRunner} JUnit or Dominion test runner or with the {@link
```

Comment:
✏️ Nit: Should there be a comma after Dominion test runner like in the next change for consistency?